src/pages/settings/Profile/ProfilePage.js

```diff
@@ -113,34 +161,53 @@ function ProfilePage(props) {
-                        <MenuItemWithTopDescription
-                            key={`${detail.title}_${index}`}
-                            shouldShowRightIcon
-                            title={detail.title}
-                            description={detail.description}
-                            onPress={() => Navigation.navigate(detail.pageRoute)}
-                            brickRoadIndicator={detail.brickRoadIndicator}
-                        />
-                    ))}
+                <View style={[styles.flex1, isSmallScreenWidth ? styles.workspaceSectionMobile : styles.workspaceSection]}>
+                    <Section
+                        title={props.translate('profilePage.publicSection.title')}
+                        subtitle={props.translate('profilePage.publicSection.subtitle')}
+                        isCentralPane
+                        subtitleMuted
+                        childrenStyles={styles.pt3}
+                        titleStyles={styles.accountSettingsSectionTitle}
+                    >
+                        {_.map(publicOptions, (detail, index) => (
+                            <MenuItemWithTopDescription
+                                key={`${detail.title}_${index}`}
+                                shouldShowRightIcon
+                                title={detail.title}
+                                description={detail.description}
+                                wrapperStyle={styles.sectionMenuItemTopDescription}
+                                onPress={() => Navigation.navigate(detail.pageRoute)}
+                                brickRoadIndicator={detail.brickRoadIndicator}
+                            />
+                        ))}
+                    </Section>
+                    <Section
+                        title={props.translate('profilePage.privateSection.title')}
+                        subtitle={props.translate('profilePage.privateSection.subtitle')}
+                        isCentralPane
+                        subtitleMuted
+                        childrenStyles={styles.pt3}
+                        titleStyles={styles.accountSettingsSectionTitle}
+                    >
+                        {isLoadingPersonalDetails ? (
+                            <FullscreenLoadingIndicator style={[styles.flex1, styles.pRelative, StyleUtils.getBackgroundColorStyle(theme.cardBG)]} />
+                        ) : (
+                            <>
+                                {_.map(privateOptions, (detail, index) => (
+                                    <MenuItemWithTopDescription
+                                        key={`${detail.title}_${index}`}
+                                        shouldShowRightIcon
+                                        title={detail.title}
+                                        description={detail.description}
+                                        wrapperStyle={styles.sectionMenuItemTopDescription}
```

Comment:
Should't we give proper margins and paddings accordingly to screen size? On figma desktop Section has padding of 32px and mobile has padding of 20px. https://www.figma.com/file/Xv3yHJooZcxRGkXbAcTZD5/%23ideal-nav-(Ideal-Nav)?type=design&node-id=3125-183280&mode=design&t=V2UcgS99inlqJWyf-0
Accordingly we should also refactor <Section/> to have 32px padding on big screens and 20px padding on small screens.

Reply:
Already implemented